docs/relnotes.md

```diff
@@ -21,61 +21,62 @@ Not published yet.
+      * `Submittable.data_nodes_being_edited()^` retrieves data nodes that are being edited
+        of a `Submittable^` entity;
+- New functions exposed by the `taipy` module: `taipy.is_deletable()^` checks if an entity can be deleted.
+  `taipy.exists()^` checks if an entity exists.
+- The encoding type of CSVDataNode and JSONDataNode can now be configured using the
+  *encoding* parameter. For more information, please refer to
+  [Configure a CSVDataNode](./manuals/core/config/data-node-config.md#csv)
+  and [Configure a JSONDataNode](./manuals/core/config/data-node-config.md#json)
+  sections.
 
-- The encoding type of CSVDataNode and JSONDataNode can now be configured using the *encoding* parameter. Please refer
-  to [Configure a CSVDataNode](./manuals/core/config/data-node-config.md#csv) and [Configure a JSONDataNode](./manuals/core/config/data-node-config.md#json)
-  sections for more information.
-
-- New abstract class:
-      * `Submittable^` to model entities that can be submitted for execution
-            The children entity classes of `Submittable` are `Scenario` and `Pipeline`;
-
-- New exposed functions:
-
-       * `taipy.exists()^` checks if an entity exists or not;
-
-       * `Submittable.get_inputs()^` retrieves input data nodes of a `Submittable` entity;
-
-       * `Submittable.get_outputs()^` retrieves output data nodes of a `Submittable` entity;
-
-       * `Submittable.get_intermediate()^` retrieves intermediate data nodes of a `Submittable` entity;
-
-       * `Submittable.is_ready_to_run()^` checks if an entity is ready to be run;
-
-       * `Submittable.data_nodes_being_edited()^` retrieves data nodes that are being edited
-            of a `Submittable^` entity;
-
-       * `is_deletable()^` checks if an entity can be deleted.
-
-- A production version of the application can now be provided with migration functions to help
-  ensure that entities from all production versions are compatible with each other.<br/>
-  For more information, please refer to [Production mode](./manuals/core/versioning/production_mode.md).
 
 <h6 style="font-size: 1.2em"><strong><code>taipy-template</code></strong></h6>
 3.0.0
 
-- A new template named "scenario-management" is available. For more information on creating a new Taipy application with the new "scenario-management" template, refer to [Create a Taipy application from a specific template](./manuals/cli/create.md#from-a-specific-template).
+- A new template named "scenario-management" is available. For more information on creating
+  a new Taipy application with the new "scenario-management" template, refer to
+  [Create a Taipy application from a specific template](./manuals/cli/create.md#from-a-specific-template).
 
 ### Improvements and changes
 
 <h6 style="font-size: 1.2em"><strong><code>taipy-core</code></strong></h6>
 3.0.0
 
+- `ScenarioConfig^` graph is now created directly from `TaskConfig^` and
```

Comment:
An article is missing (as well as in the items below).
  "A ScenarioConfig"...
('graph' may be confusing with graphics, but hey)